package.json

```diff
@@ -14,10 +14,14 @@
     "node": "^12"
   },
   "scripts": {
-    "test": "true"
+    "test": "true",
+    "semrel-releaser": "lerna exec --scope=monorepo-releaser yarn semantic-release --stream",
```

Comment:
I thought we weren't planning on actually releasing this since it's only used by this repo. Is there a reason why that changed?

Reply:
Once I built it, it seemed odd to include the public-friendly GHE settings with no way to use them in GHE.
Also...kinda the point of this package is to make releases easy and allow modules to not have any other dependency for releasing besides this one. Stated purpose of this repo was partially to reduce devOps config/setup in ET and CST. But since those aren't monorepos, seems like overkill to use them there